### ISSUE TITLE
训练速度问题

### ISSUE THREAD

**Issue by lao-xu** [PaddlePaddle/PaddleDetection#2778]
旧版本ssd训练12w轮，相当于120epoch，需要8-9小时
新版本ssd训练120epoch，却需要17小时多，这速度差的太远了，不知道是什么原因，不应该差这么多啊
[04/27 15:36:53] ppdet.engine INFO: Epoch: [0] [   0/1034] learning_rate: 0.000333 loss: 26.950333 eta: 16:56:21 batch_cost: 0.4915 data_cost: 0.0022 ips: 32.5557 images/s
[04/27 15:37:03] ppdet.engine INFO: Epoch: [0] [  20/1034] learning_rate: 0.000360 loss: 17.336845 eta: 17:22:26 batch_cost: 0.5048 data_cost: 0.0017 ips: 31.6959 images/s
[04/27 15:37:13] ppdet.engine INFO: Epoch: [0] [  40/1034] learning_rate: 0.000387 loss: 12.285319 eta: 17:25:59 batch_cost: 0.5079 data_cost: 0.0027 ips: 31.5053 images/s
[04/27 15:37:23] ppdet.engine INFO: Epoch: [0] [  60/1034] learning_rate: 0.000413 loss: 9.443251 eta: 17:21:53 batch_cost: 0

**Comment by jerrywgz** [comment]
https://github.com/PaddlePaddle/PaddleDetection/pull/2761 目前我们已经基于Paddle develop版本对ssd模型训练速度做了优化，待pr merge后动态图速度可以超过静态图

**Comment by lao-xu** [comment]
太好了，不知道何时能放出来，我们正在基于detection做开发，希望能快点

**Comment by jerrywgz** [comment]
预计本周可以合入

**Comment by lao-xu** [comment]
> 预计本周可以合入

好，并入后请告知下，谢谢
